diff --git a/textual-inversion/textual_inversion.py b/textual-inversion/textual_inversion.py
index a18db4a..89e0d37 100644
--- a/textual-inversion/textual_inversion.py
+++ b/textual-inversion/textual_inversion.py
@@ -575,7 +575,7 @@ def main():
         gradient_accumulation_steps=args.gradient_accumulation_steps,
         mixed_precision=args.mixed_precision,
         log_with=args.report_to,
-        logging_dir=logging_dir,
+        project_dir=logging_dir,
         project_config=accelerator_project_config,
     )
 
