diff --git a/lib/core/config.py b/lib/core/config.py
index 2620d15..4a7b2ef 100755
--- a/lib/core/config.py
+++ b/lib/core/config.py
@@ -511,7 +511,7 @@ def assert_and_infer_cfg(make_immutable=True):
 def merge_cfg_from_file(cfg_filename):
     """Load a yaml config file and merge it into the global config."""
     with open(cfg_filename, 'r') as f:
-        yaml_cfg = AttrDict(yaml.load(f))
+        yaml_cfg = AttrDict(yaml.load(f, Loader=yaml.SafeLoader))
     _merge_a_into_b(yaml_cfg, __C)
 
 cfg_from_file = merge_cfg_from_file
